src/lib/containers/MarkdownSynapse.tsx

```diff
@@ -262,13 +262,18 @@ export default class MarkdownSynapse extends React.Component<
       '[id^="mathjax-"]',
     )
     // go through all obtained elements and transform them with katex
+    const regEx = new RegExp(/\\[()[\]]/, 'g') // Look for a '\' followed by either '(', ')', '[', or ']'. We delete these strings since they interfere with katex processing.
```

Comment:
This syntax is a way to indicate that the math should be display inline, but we handle this outside of the latex (in the markdown it's either an inline math element or a block element).
https://www.overleaf.com/learn/latex/mathematical_expressions